test/views/benefits/income/IncurredCostsBenefitsAmountViewSpec.scala

```diff
@@ -151,7 +148,7 @@ class IncurredCostsBenefitsAmountViewSpec extends ViewUnitTest {
         titleCheck(userScenario.specificExpectedResults.get.expectedTitle, userScenario.isWelsh)
         h1Check(userScenario.specificExpectedResults.get.expectedHeading)
         captionCheck(expectedCaption)
-        textOnPageCheck(enterTotalText, paragraphSelector2)
+        textOnPageCheck(enterTotalText, paragraphSelector(2))
```

Comment:
It seems `paragraphSelector2` is not used anymore in the file and we can delete it.

Reply:
It has been removed.